secator/query/mongodb.py

```
@@ -54,6 +54,23 @@ def _execute_search(self, query: dict, limit: int = 100, exclude_fields: list =
+			client = self._get_client()
+			db = client.main
+			batch = []
+			for doc in db.findings.find(query).batch_size(batch_size):
+				doc.pop('_id', None)
+				batch.append(doc)
+				if len(batch) >= batch_size:
+					yield batch
+					batch = []
+			if batch:
+				yield batch
+		except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)